librz/include/rz_types.h

```diff
@@ -675,6 +675,13 @@ static inline void rz_run_call10(void *fcn, void *arg1, void *arg2, void *arg3,
 		(arg1, arg2, arg3, arg4, arg5, arg6, arg7, arg8, arg9, arg10);
 }
 
+#define RZ_V_NOT(op, fail_ret) \
+	if ((op) == (fail_ret)) \
+		eprintf (#op" at %s:%d failed: %s\n", __FILE__, __LINE__, strerror (errno))
```

Comment:
shall we use `eprintf` or RZ_LOG_WARN/RZ_LOG_INFO ? Are these eprintf captures during rz-test? Because I think RZ_LOG_WARN is captured and at least we can know if something fail in CI

Reply:
d8abe061c1b98cdc73f3cf58131088c4712e66ff emm i think you know better than me